packages/app/src/components/time-series-chart/logic/use-value-width.ts

```diff
@@ -0,0 +1,26 @@
+import { useMemo } from 'react';
+import { SeriesConfig } from './series';
+import { useStringFormatting } from './use-string-formatting';
+import { TimestampedValue } from '@corona-dashboard/common';
+
+export function useValueWidth<T extends TimestampedValue>(
+  values: T[],
+  seriesConfig: SeriesConfig<T>,
+  isPercentage: boolean | undefined
+) {
+  const { formatSeriesValue } = useStringFormatting();
+  const valueMaxWidth = useMemo(() => {
+    const valueLengths: number[] = [];
+    seriesConfig.forEach((config) => {
+      return valueLengths.push(
+        ...values.map(
+          (value) => formatSeriesValue(value, config, isPercentage).length
+        )
+      );
+    });
```

Comment:
nit; not sure the impact on performance, but this can create a very long array. We could call Math.max more often;
```suggestion
    const valueLengths: number[] = [];
    seriesConfig.forEach((config) => {
      return valueLengths.push(
        Math.max(...values.map(
          (value) => formatSeriesValue(value, config, isPercentage).length
        ))
      );
    });
```